src/tests/stdlib/file_exist.ab

```diff
@@ -0,0 +1,12 @@
+import * from "std"
+main {
+    let tmpdir = unsafe $mktemp -d /tmp/amber-XXXX$
+    unsafe $cd {tmpdir}$
+    unsafe $touch test.txt$
```

Comment:
I just realized that once we merge https://github.com/Ph0enixKM/Amber/pull/260 then we could add a function `create_file(path, content = "")` This would replace the `touch` here with `create_file("test.txt")`

Reply:
I used touch in this way the tests are not so much dependent each other in some cases.